Scripts/RemediationScripts/Remediate-SetServiceBusMinReqTLSVersion.ps1

```diff
@@ -0,0 +1,779 @@
+        
+        To know more about the options supported by the remediation command, execute:
+        
+        Get-Help Set-SecureTLSVersionForServiceBusNamespaces -Detailed
+
+    To roll back:
+
+        1. Set TLS version for Azure Service Bus Namespace(s) in the Subscription, from a previously taken snapshot:
+        
+            Reset-SecureTLSVersionForServiceBusNamespaces -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -RollbackFilePath C:\AzTS\Subscriptions\00000000-xxxx-0000-xxxx-000000000000\202109131040\SetSecureTLSVersionForServiceBusNamespaces\ServiceBusNamespacesBackUp.csv
+       
+        To know more about the options supported by the roll back command, execute:
+        
+        Get-Help Reset-SecureTLSVersionForServiceBusNamespaces -Detailed        
+###>
+
+
+function Setup-Prerequisites {
+    <#
+        .SYNOPSIS
+        Checks if the prerequisites are met, else, sets them up.
+
+        .DESCRIPTION
+        Checks if the prerequisites are met, else, sets them up.
+        Includes installing any required Azure modules.
+
+        .INPUTS
+        None. You cannot pipe objects to Setup-Prerequisites.
+
+        .OUTPUTS
+        None. Setup-Prerequisites does not return anything that can be piped and used as an input to another command.
+
+        .EXAMPLE
+        PS> Setup-Prerequisites
+
+        .LINK
+        None
+    #>
+
+    # List of required modules
+    $requiredModules = @("Az.Accounts", "Az.ServiceBus")
+
+    Write-Host "Required modules: $($requiredModules -join ', ')"
+    Write-Host $([Constants]::SingleDashLine)    
+    Write-Host "Checking if the required modules are present..." -ForegroundColor $([Constants]::MessageType.Info)
+    Write-Host $([Constants]::SingleDashLine)
+
+    $availableModules = $(Get-Module -ListAvailable $requiredModules -ErrorAction Stop)
+
+    # Check if the required modules are installed.
+    $requiredModules | ForEach-Object {
+        if ($availableModules.Name -notcontains $_) {
+            Write-Host "$($_) module is not present." -ForegroundColor $([Constants]::MessageType.Warning)
+            Write-Host "Installing [$($_)] module..." -ForegroundColor $([Constants]::MessageType.Info)
+            Install-Module -Name $_ -Scope CurrentUser -Repository 'PSGallery' -ErrorAction Stop
+            Write-Host "[$($_)] module installed." -ForegroundColor $([Constants]::MessageType.Update)
+        }
+        else {
+            Write-Host "[$($_)] module is present." -ForegroundColor $([Constants]::MessageType.Update)
+        }
+    }
+    Write-Host $([Constants]::SingleDashLine)
+}
+
+function Set-SecureTLSVersionForServiceBusNamespaces {
+    <#
+        .SYNOPSIS
```

Comment:
Add parameter details, I can see InputFilePath, Autoremediation, TimeStamp is missing.

Reply:
Done.